api/src/main/java/org/openmrs/module/pihcore/apploader/CustomAppLoaderFactory.java

```diff
@@ -2195,7 +2195,7 @@ private void enableMCHGainNewborn() {
                     "link",
                     enterStandardHtmlFormLink(PihCoreUtil.getFormResource("gainNewbornRegister.xml")),
                     PihEmrConfigConstants.PRIVILEGE_TASK_EMR_ENTER_MCH,
-                    and(sessionLocationHasTag("Maternal and Child Location"),
```

Comment:
Tangent: can we potentially removed the "Maternal and Child Location", or is it still being used for something else.